page/about.html

```diff
@@ -0,0 +1,231 @@
+                    Contact Us
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#logo"  class="mob-menu-item" id="sponsor">
+                    Current Logo 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#past-events"  class="mob-menu-item" id="join">
+                    Past summits 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#partner"  class="mob-menu-item" id="news">
+                    Partner 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="about-intro-container">
+        <header>
+            <ul class="most-top">
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-facebook fa-fw"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-twitter fa-fw"></i>
+                    </a>
+                </li>
+                <li class="icon">English</li>
+                <li>
+                    <a href="#" class="icon">My Page</a>
+                </li>
+                <li>
+                    <a href="#" class="icon">Logout</a>
+                </li>
+            </ul>
+            <span class="mobile-menu mob-side-nav">
+                <img src="../assets/icons/menuIcon.svg" alt="Humburgur Menu">
+            </span>
+    
+        </header>
+
+        <section class="home-intro">
+
+            <div class="page-intro">
+                <h1 id="title-desc">International Technology Talk Summit 2022</h1>
+                <div class="opening-desc">
+                    <p>The Global summit brings together the community of experts, academics adn activists who comprise the Creative Commmons affilate network in a different country every tow this year, Wehope to expand our inivitation list including organizationa ans individiduals.</p>
+                </div>
+            </div>
+            
+        </section>
+
+    </section>
+
+    <section class="contact-us" id="contact-us">
+        <p>Please contact us per Email for any further questions about CC Global Summit</p>
+        <a>bush7840@yahoo.com</a>
+    </section>
+
+    <section class="logo" id="logo">
+        <h1>Creative commons Global Summit 2015 Logo</h1>
+        <p>The logo of CC global summit 2015 was decided through the logo competition from 8.,june to 7,July.</p>
+        <img src="../assets/images/company_image.png" alt="">
+    </section>
+
```

Comment:
- [ ] I think you should have a `divider` for the header of this section. The `divider` is the _**orange horizontal line**_ under the header of this section. You can adopt the same idea that you used for the `divider` of the `Home Page`.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/153716563-5dec3830-2fd8-4d3f-a4b2-f8be3700ab60.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/153716589-77b5f3b4-9047-4167-b8eb-416c4ca4b087.png)

Reply:
solved as per your request